pkg/image/registry/imagestreamimport/strategy.go

```diff
@@ -43,7 +45,13 @@ func (s *strategy) PrepareImageForCreate(obj runtime.Object) {
 	image.DockerImageConfig = ""
 }
 
+// SetAllowedImportRegistries sets the list of registries allowed to import the images
+// from that is specified in the master config.
+func (s *strategy) SetAllowedImportRegistries(registries *serverapi.AllowedRegistries) {
+	s.allowedRegistries = registries
+}
+
 func (s *strategy) Validate(ctx kapi.Context, obj runtime.Object) field.ErrorList {
 	isi := obj.(*api.ImageStreamImport)
-	return validation.ValidateImageStreamImport(isi)
+	return validation.ValidateImageStreamImport(isi, s.allowedRegistries)
```

Comment:
Any admin who can create images or set image stream mappings should be able to bypass this check.  Move this out of validation and do the ACL check in PrepareCreate and PrepareUpdate